app/src/main/java/com/github/steroidteam/todolist/database/FirebaseDatabase.java

```diff
@@ -146,4 +146,12 @@ public Task getTask(@NonNull UUID todoListID, @NonNull Integer taskIndex)
             throw new DatabaseException(e.toString());
         }
     }
+
+    @Override
+    public void assignMainImage(UUID noteID, String path) {}
+
+    @Override
+    public String getMainImagePath(UUID noteID) {
+        return null;
```

Comment:
maybe you can return `throw new UnsupportedOperationException("TODO");`
So we don't forget to implement it